pcdsdevices/pc.py

```diff
@@ -4,12 +4,21 @@
 from ophyd import Component as Cpt
 from ophyd import Device
 
+from .analog_signals import FDQ
 from .digital_signals import J120K
 
 
 class PhotonCollimator(Device):
     """
-    Photon Collimator Stub for Cooling Readback
+    Photon Collimator Stub for Cooling Switch Readback
```

Comment:
Is a stub referring to the type of device?  Or is it saying that this python class is a stub device (not really standard terminology, but it made me think of a class that's meant to be subclassed). 

Also these class names imply to me that `PhotonCollimatorFDQ` is a child of `PhotonCollimator`.  The `PhotonCollimatorFDQ` device doesn't seem to have any components related to a collimator at all, just the FDQ.  is there a better name fore this device?  `PCWFlowMeter`?

Reply:
The reason I used the word stub is because I've created this photon collimator class with nothing in it other than the flow sensors readout; it's superfluous. I will delete it. I agree that it sounds inherited, but its not. I can think of one good reason here to actually inherit in FDQ; if later someone wants to add functionality, they can update the base `PhotonCollimator` class only.